packages/frames.js/src/getFrameFlattened.ts

```diff
@@ -10,7 +10,7 @@ export function getFrameFlattened(frame: Frame): FrameFlattened {
     frame.accepts && Boolean(frame.accepts.length)
```

Comment:
@davidfurlong @stephancill this line now makes all frames rendered by new api incompatible with Open Frames spec, since we don't provide `accepts` anywhere. That also makes sense since `openframes` middleware is optional.

I think the current behaviour where examples fail on open frames validation is not correct  since we have `farcaster` middleware used by default, that means we should also support it automatically, right?

Reply:
See:

<img width="2672" alt="image" src="https://github.com/framesjs/frames.js/assets/174716/baa461fc-b928-4054-ac40-6aea40cd1ed8">
<img width="2672" alt="image" src="https://github.com/framesjs/frames.js/assets/174716/14195b0b-cc42-4606-9950-fd7a73f34c89">

Reply:
I'm confused because `openframes` middleware is optional, it is not used by default. Somehow it makes sense to me, that we should not output `of:*` meta tags, unless the middleware is used. That somehow makes sense to me personally but I probably don't get it. @davidfurlong @stephancill

Reply:
I agree with your conclusion. I think this is a problem with our implementation treating farcaster as a first class citizen a level above open frames

If we refactor the payload parsing to require the user to explicitly specify farcaster then we can more confidently include open frames tags automatically because we can accurately deduce when a frame doesn't make use of farcaster context

First class, automatic open frames support can be a framework enhancement in a separate ticket imo